.github/workflows/manylinux_2014.yml

```diff
@@ -159,7 +159,22 @@ jobs:
             ${{ fromJSON(needs.docker.outputs.images).ov_build.manylinux2014_x86_64 }} \
             /bin/bash -c "
               python3.12 -m pip install -r /work/src/src/bindings/python/wheel/requirements-dev.txt
-              cmake -DPython3_EXECUTABLE=/usr/local/bin/python3.12 -DENABLE_CPPLINT=OFF -DENABLE_NCC_STYLE=OFF -DENABLE_TESTS=ON -DCMAKE_VERBOSE_MAKEFILE=ON -DENABLE_OV_TF_FRONTEND=OFF -DENABLE_OV_TF_LITE_FRONTEND=OFF -DENABLE_OV_PADDLE_FRONTEND=OFF -DENABLE_OV_PYTORCH_FRONTEND=ON -DENABLE_OV_JAX_FRONTEND=OFF -DENABLE_OV_ONNX_FRONTEND=ON -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON -S /work/src -B /work/build &&
+              cmake \
+                -DPython3_EXECUTABLE=/usr/local/bin/python3.12 \
+                -DENABLE_CPPLINT=OFF \
+                -DENABLE_NCC_STYLE=OFF \
+                -DENABLE_TESTS=ON \
+                -DCMAKE_VERBOSE_MAKEFILE=ON \
+                -DENABLE_OV_TF_FRONTEND=ON \
+                -DENABLE_OV_TF_LITE_FRONTEND=ON \
+                -DENABLE_OV_PADDLE_FRONTEND=ON \
+                -DENABLE_OV_PYTORCH_FRONTEND=ON \
+                -DENABLE_OV_JAX_FRONTEND=OFF \
+                -DENABLE_OV_ONNX_FRONTEND=ON \
```

Comment:
all frontends are enabled by default, just drop explicit options except `DENABLE_OV_JAX_FRONTEND=OFF`

Reply:
we should not depends on default values

Reply:
default options ensure that you can build the same product as Release archive on S3.

I think we need to use this rule and avoid listing all options manually.

Reply:
That is the reason to set them explicitly) to avoid difference between Jenkins and GHA builds